readium/navigator/src/main/java/org/readium/r2/navigator/media3/exoplayer/ExoPlayerPreferencesFilters.kt

```diff
@@ -16,7 +16,7 @@ import org.readium.r2.shared.ExperimentalReadiumApi
 object ExoPlayerSharedPreferencesFilter : PreferencesFilter<ExoPlayerPreferences> {
```

Comment:
Are these really useful for ExoPlayer? Do you expect that we will have publication-specific preferences in the audio engine?

Reply:
You're right, let's forget about that for now.